db/seeds.rb

```diff
@@ -77,6 +77,20 @@ def get_country
     )
 
     puts 'Created default projects'
+
+    opensource = u.courses.create!(
```

Comment:
I see we have an object called opensource but I can't seem to find where this is referenced

Reply:
Fixed.